### ISSUE TITLE
Add the plan gist to crdb_internal.cluster_queries

### ISSUE THREAD

**Issue by kevin-v-ngo** [cockroachdb/cockroach#76554]
We are introducing the plan gist to statement statistics to allow us to capture all plans of a statement fingerprint historically. We should add the plan gist as a column crdb_internal.cluster_queries for active statements. This will allow users to troubleshoot slow running or 'hung' queries for potential suboptimal or regressed plans in real-time.

**Comment by kevin-v-ngo** [comment]
Once this is supported, we can add the EXPLAIN plan to active executions as well. Even more useful once we have recently executed executions.

**Comment by kevin-v-ngo** [comment]
Hi @xinhaoz, do we have an issue tracking adding the plan itself to active (and recently) executions?

**Comment by xinhaoz** [comment]
> server, sql: add plan gist to active queries #88770

Yup, and it's been closed: https://github.com/cockroachdb/cockroach/issues/79128
